util/gitutil/git_cli.go

```diff
@@ -119,20 +119,13 @@ func NewGitCLI(opts ...Option) *GitCLI {
 // New returns a new git client with the same config as the current one, but
 // with the given options applied on top.
 func (cli *GitCLI) New(opts ...Option) *GitCLI {
```

Comment:
Maybe this should be renamed to `WithOptions()` or smth. to make it clearer that this is a constructor but a way to make a new variant of the existing instance with additional config options added.